src/components/date/__internal__/date-formats/date-formats.spec.ts

```diff
@@ -511,7 +511,22 @@ describe.each([...euLocales, ...naLocales, ...cnLocales])(
     });
 
     it("the expected format", () => {
-      expect(format).toEqual(formatMap[locale]);
+      const { format } = getFormatData({ code: locale });
+
+      expect(format).toEqual(formatMap[locale as keyof typeof formatMap]);
+    });
+
+    it("defaults to en-GB locale if no localecode string passed", () => {
```

Comment:
question/nitpick: is it meant to be `locale code` or is `localecode` correct?

Reply:
probably either `local code` or `localCode`, I'll update accordingly